### PR TITLE
Set schema to array of items instead of single item [SA-15481] develop

### PR DESCRIPTION
**Pull request by tomcdonnell**
### Documentation before change:
image

### Documentation after change:
image

### Actual response:
image

### Documentation before change:
image

### Documentation after change:
image
image

### Actual response:
image

### CODE REVIEW

**Comment by armatronic** [comment]
There's still something strange showing in the API docs for groups, the `[any-key]` key is still displaying.
image

I don't think that should be there at all, and instead all the other object properties should be moved up into the array's item.
image

(I _think_ that's what's going on? I can't be sure, but the `[any-key]` implies that the groups is still a list of objects.)

File: openapi/components/schemas/user-readableFields.yaml [hunk 1]
```diff
@@ -17,16 +17,18 @@ properties:
   groups:
```
Comment (Brahmah) [inline openapi/components/schemas/user-readableFields.yaml:17]: Groups should look something like:
```
  groups:
    description: |
      This user's group memberships.
    type: array
    items:
      type: object
      title: Group Membership
      description: |
          A group within Schoolbox.
          
          This contains some of the folder's fields in this context.
      properties:
          $ref: ../schemas/userGroupMembership-readableFields.yaml
``` 
Don't see why `additionalProperties` is needed here alas, this is just after a quick look out of context.

Docs will look like:
image

Comment (tomcdonnell) [reply openapi/components/schemas/user-readableFields.yaml:17]: Thanks @Brahmah, I agree.  Fixed.

**Comment by tomcdonnell** [comment]
> There's still something strange showing in the API docs for groups, the `[any-key]` key is still displaying.

This has been fixed.

**Pull request merged**